server/server.js

```diff
@@ -1,52 +1,20 @@
 import express from "express";
-import path from "path";
-import devMode from "./dev-mode";
-import ComputeHandler from "./compute-handler";
-import responseTime from "response-time";
-import updateUser from "./user-update";
 
-module.exports = function Server(options = {}) {
-  const { port, Hull, devMode: dev, hostSecret } = options;
-  const { BatchHandler, NotifHandler, Routes, Middleware } = Hull;
-  const { Readme, Manifest } = Routes;
+import ComputeHandler from "./actions/compute-handler";
+import NotifyHandler from "./actions/notify-handler";
+import devMode from "./dev-mode";
 
+module.exports = function Server(connector, options = {}) {
   const app = express();
+  const { Hull, hostSecret } = options;
 
-  if (dev) app.use(devMode());
-  app.use(responseTime());
-  app.use(express.static(path.resolve(__dirname, "..", "dist")));
-  app.use(express.static(path.resolve(__dirname, "..", "assets")));
-
-  app.set("views", path.resolve(__dirname, "..", "views"));
-
-  app.get("/manifest.json", Manifest(__dirname));
-  app.get("/", Readme);
-  app.get("/readme", Readme);
+  app.post("/compute", ComputeHandler({ hostSecret, connector, Hull }));
 
+  if (options.devMode) app.use(devMode());
+  connector.setupApp(app);
```

Comment:
We need to define the `/compute` endpoint before applying middlewares for some reason..